src/routes/scheduledItems.integration.ts

```diff
@@ -0,0 +1,95 @@
+import { expect } from 'chai';
+import { app } from '../main';
+import request from 'supertest';
+import config from '../config';
+import { stories } from './scheduledItems';
+
+describe('/scheduled-items/:scheduledSurfaceID?date=date&apikey=apikey', () => {
+  const requestAgent = request.agent(app);
+
+  const testNewTab = 'POCKET_HITS_EN_US';
+  const testDate = '2050-01-01';
+  const validUrl = `/scheduled-items/${testNewTab}?date=${testDate}&apikey=${config.aws.brazeApiKey}`;
+
+  const testStories = {
+    stories: [
+      {
+        id: '123-abc',
+        url: 'www.test-url.com',
+        title: 'test-title',
+        excerpt: 'test-excerpt',
+        imageUrl: 'www.test-image-url.com',
+        authors: 'test-author',
+        publisher: 'test-publisher',
+      },
+      {
+        id: '456-cde',
+        url: 'www.second-test-url.com',
+        title: 'second-test-title',
+        excerpt: 'second-test-excerpt',
+        imageUrl: 'www.second-test-image-url.com',
+        authors: 'second-test-author',
+        publisher: 'second-test-publisher',
+      },
+    ],
+  };
+
+  it('should return 200 OK and correct headers when valid query params are provided', async () => {
+    const response = await requestAgent.get(validUrl);
+
+    expect(response.statusCode).equals(200);
+    // checking if the cache-control header has been set correctly
+    expect(response.headers['cache-control']).to.not.be.undefined;
+    expect(response.headers['cache-control']).to.equal('public, max-age=120');
+  });
+
+  it('should return correct data when valid query params are provided', async () => {
+    // spying on the getStories function to make it return a mock response
+    jest.spyOn(stories, 'getStories').mockResolvedValue(testStories as any);
```

Comment:
existing test
only change is mock stories.getStories as the router and getstories were in same file and mock didnt work

and changed the `requestAgent` to call `request(app).get`